src/v3/src/components/Link/Link.tsx

```diff
@@ -63,12 +63,19 @@ const Link: UISchemaElementComponent<{
   return (
     typeof href === 'undefined' ? (
       <LinkMui
-        // eslint-disable-next-line no-script-url
-        href="javascript:void(0)"
+        component="button"
+        variant="body1"
+        role="link"
         onClick={onClick}
         aria-describedby={ariaDescribedBy}
         ref={focusRef}
         data-se={dataSe}
+        sx={{
+          '&:hover': {
+            cursor: 'pointer',
+          },
+          verticalAlign: 'baseline',
```

Comment:
I know sometime a while back we used verticalAlign on a link and it caused this view to look uneven, I assume that is not the case anymore? I think it should be fine, because last time we used `middle` and that caused some issues, so I think `baseline` is ok. 
<img width="518" alt="image" src="https://user-images.githubusercontent.com/97472729/226485388-768a6730-8e12-4a68-bf8e-e24bdbd1fc36.png">

Reply:
Yeah, the "Sign up" text was appearing slightly below the rest of the text (~3px) without any extra styling. When I put `baseline` it appears correctly.